js/pro/bitfinex.js

```diff
@@ -477,6 +479,7 @@ module.exports = class bitfinex extends bitfinexRest {
          */
         await this.loadMarkets ();
         await this.authenticate ();
+        symbol = this.symbol (symbol);
```

Comment:
@pcriadoperez this will crash if `symbol` is undefined, we need to add a check

Reply:
great to know. I assume `this.symbol()` would just return undefined in that case. I'll fix it